molecule/default/molecule.yml

```diff
@@ -11,6 +11,10 @@ driver:
 platforms:
   - name: aws-dynamo-db-trusty
     image: ubuntu:trusty
```

Comment:
I think we should remove Trusty from here.

Reply:
Sure, I'll remove